py/build.gradle

```diff
@@ -127,7 +127,7 @@ def copyPySetup() {
         filter {
             it.replaceAll("SUBST_SPARK_MAJOR_VERSION", sparkMajorVersion)
                     .replaceAll("SUBST_SPARK_VERSION", sparkVersion)
-                    .replaceAll("SUBST_SW_VERSION", version.substring(0, version.lastIndexOf("-")))
+                    .replaceAll("SUBST_SW_VERSION", version.substring(0, version.lastIndexOf("-").replace("_")))
```

Comment:
Shouldn't it be the same as [here](https://github.com/h2oai/sparkling-water/pull/1585/files#diff-bcffd8069a181d6a460a3178647d82d7R169)? or can you give me more details why it's like that?

Reply:
Yup, should be the same